build-info-extractor/src/testFixtures/java/org/jfrog/build/IntegrationTestsBase.java

```diff
@@ -210,30 +177,12 @@ protected void createTestRepo(String repo) throws IOException {
      * @throws IOException
      */
     protected void deleteTestRepo(String repo) throws IOException {
-        if (StringUtils.isBlank(repo)) {
-            return;
-        }
-        HttpRequestBase httpRequest = new HttpDelete(getRepoApiUrl(repo));
-        try (CloseableHttpResponse response = preemptiveHttpClient.execute(httpRequest)) {
-            EntityUtils.consumeQuietly(response.getEntity());
-            int statusCode = response.getStatusLine().getStatusCode();
-            if (statusCode != HttpStatus.SC_OK) {
-                throw new IOException(getRepositoryCustomErrorMessage(repo, false, response.getStatusLine()));
-            }
-        }
-    }
-
-    private String getRepoApiUrl(String repo) {
-        String fullItemUrl = url + StringUtils.join(new String[]{API_REPOSITORIES, repo}, "/");
-        return ArtifactoryHttpClient.encodeUrl(fullItemUrl);
+        artifactoryManager.deleteRepository(repo);
     }
 
     /**
      * Read spec file and replace the placeholder test data.
      *
```

Comment:
```suggestion
    /**
     * Read spec file and replace the placeholder test data.
     *
     * @param specFile      - the spec file
     * @param workSpacePath - workspace path
     * @return the File Spec as a string.
     * @throws IOException in case of any I/O error.
     */
```